app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -63,7 +63,8 @@ def create_usps_enrollment(enrollment, pii)
             city: transliterate(pii[:city]),
             state: pii[:state],
             zip_code: pii[:zipcode],
-            email: 'no-reply@login.gov',
+            email: IdentityConfig.store.usps_ipp_enrollment_status_update_email_address.presence ||
+              'no-reply@login.gov',
```

Comment:
The fallback isn't necessary with the fallback configuration in place (though you'll need to update the test if you accept this change).

```suggestion
            email: IdentityConfig.store.usps_ipp_enrollment_status_update_email_address,
```

Reply:
I was on the fence about it myself -- it's only useful if someone accidentally edits the config and specifies an empty string for the email address. And even then it will be a kind of sneaky intervention. Yet it's still a pretty common pattern ([example](https://github.com/18F/identity-idp/blob/f1010f7cd640a3508cb3fbb39105e31abaeed99b/app/jobs/get_usps_proofing_results_job.rb#L476)) for there to be a hard-coded default on top of the configuration default.

I'll go ahead with your suggestion. Also going to start a thread in slack to hear other folks' thoughts on if there's a good reason to have the hard-coded default